build_utils/verify_tsconfig_references.js

```diff
@@ -82,12 +106,11 @@ const main = async () => {
     ),
     v => v.length,
   )
-
-  const workspacePackageToReference = (
-    package, refPackage,
-  ) => ({
-    path: path.relative(workspaces[package].location, workspaces[refPackage].location)
-  })
+  
+  const workspacePackageToReference = (package, refPackage) => {
+    const relativePath = path.relative(workspaces[package].location, workspaces[refPackage])
```

Comment:
the old code was correct, this seems to be broken.
either revert this change, or fix it
```suggestion
    const relativePath = path.relative(workspaces[package].location, workspaces[refPackage].location)
```